mindsdb/api/mysql/mysql_proxy/executor/executor_commands.py

```diff
@@ -550,7 +554,10 @@ def execute_command(self, statement):
             db_name = statement.value.parts[-1]
             self.change_default_db(db_name)
             return ExecuteAnswer(ANSWER_TYPE.OK)
-        elif type(statement) == CreatePredictor:
+        elif type(statement) in (
```

Comment:
maybe isinstance(..., CreatePredictorBase) ?
and put 'elif type(statement) == FinetunePredictor' upper
it is not important, just a proposal

Reply:
Will keep as is for now, otherwise there is a small risk that the `if` order may be switched around mistakenly and so breaking either finetuning or creation